e2e/LongHaul/device/IotHub.cs

```diff
@@ -169,10 +169,13 @@ public async Task ReportReadOnlyPropertiesAsync(Logger logger, CancellationToken
                     {
                         { "TotalTelemetryMessagesSent", _totalTelemetryMessagesSent },
                     };
+                    var sw = Stopwatch.StartNew();
```

Comment:
You can create the stopwatch above, and then use Reset and Stop in the loop.

That way the object isn't created over and over again.